reference/extensions/command_wrapper.rst

```diff
@@ -0,0 +1,33 @@
+.. _reference_extensions_command_wrapper:
+
+Command wrapper
+---------------
+
+The ``cmd_wrapper.py`` extension plugin is a Python script that receives the command line
+argument provided by ``self.run()`` recipe calls, and allows intercepting them and returning
+a new one. 
+
+This plugin must be located in the ``extensions/plugins`` cache folder, and can be installed
+with the ``conan config install`` command.
+
+For example:
+
+.. code-block:: python
+
+    def cmd_wrapper(cmd):
+        return 'echo "{}"'.format(cmd)
+
+Would just intercept the commands and display them to terminal, which means that all commmands
+in all recipes ``self.run()`` will not execute, but just be echoed.
```

Comment:
that's `--dry-run`, more or less, but what if I want both execute and display them into the terminal?